test/dockerpy/README.md

```diff
@@ -0,0 +1,35 @@
+# Dockerpy regression test
+
+Python test suite to validate Podman endpoints using dockerpy library
+
+Running tests
+=============
+To run the tests locally in your sandbox:
+
+#### Make sure that the Podman system service is running to do so
+
+```
+sudo podman --log-level=debug system service -t0 unix://var/run/podman.sock
```

Comment:
The default socket path is `"unix:/run/podman/podman.sock"`. I suggest to keep using that.

Reply:
done